README.md

```diff
@@ -172,7 +172,7 @@ The policy consists of a set of rules. Each rule starts with a pattern that need
 In addition to the pattern, rules can also contain a `verify` flag which specifies whether the image requires a signature or can skip verification. If `verify` is set to `true`, a valid signature is required; otherwise, the image is admitted without verification. The default value for `verify` is `true`. Additionally, a rule can have `delegations` which correspond to Notary delegation roles. These are to be understood as specific signers that need to be present in the Notary trust data in order for signature verification to be successful. Lastly a `notary` and `key` field can be added, which reference which of the notaries should be consulted when accessing signatures for a given image, and which of the keys should be used to verify these signatures. Make sure that the `notary` field references one of the notaries `name` fields and the `key` field references of the public root keys `name` field, within the given notary configuration. If no `notary` or `key` fields are given, Connaisseur will either do one of two things:
 
 1. Should there only be one notary configuration or only one key defined within a given configuration, this configuration or key will be taken by default for verification purposes.
-2. Should there be more then one notary configuration or key, Connaisseur assumes that one of those has the name "default" and take this one by default.
+2. Should there be more then one notary configuration or key, Connaisseur assumes that one of those has the `name` field set to `default` and take this one by default.
```

Comment:
```suggestion
2. Should there be more then one notary configuration or key and if one of the `name` fields is set to `default`, Connaisseur will take this one. If no default is specified and no specific entry defined, Connaisseur will abort the validation with an error.
```

I'd like this to be a bit more specific, along the lines of the above, if that's fine with you.

Reply:
Would then do the same change in the setup/README